implementations/rust/ockam/ockam_command/src/node/create.rs

```diff
@@ -347,32 +346,36 @@ async fn run_foreground_node(
         }
     }
 
+    // Create a channel for communicating back to the main thread
+    let (tx, mut rx) = tokio::sync::mpsc::channel(2);
+
+    // Register a handler for SIGINT, SIGTERM, SIGHUP
+    let tx_clone = tx.clone();
+    ctrlc::set_handler(move || {
+        tx_clone.blocking_send(()).unwrap();
```

Comment:
Add message here:

```rust
opts.shell.write_line(format!(
    "{} Ctrl+C signal received",
    "!".light_yellow(),
    &cmd.node_name
))?;
```

Reply:
>  I wasn't sure if we want the node_name printed as well. let me know.

Sorry, copy-paste error. No need to show the node_name. I added a separate comment about the unwrapped `write_line()`.